api/config/cron-tasks.js

```diff
@@ -7,26 +7,27 @@ module.exports = {
     ];
 
     contentTypes.map(async (type) => {
-      const draftArticlesToPublish = await strapi.db.query(type).findMany({
-        where: {
-          ready: true,
-          publishedAt: null,
-          publish_at: {
-            $lte: new Date(),
+      try {
+        const draftArticlesToPublish = await strapi.db.query(type).findMany({
```

Comment:
Since this is a demo, we really should switch this to using entityService since that is what we recommend:

![image](https://user-images.githubusercontent.com/8593673/193296235-44c959bc-6228-4b08-b119-01d5a9c283f4.png)

Reply:
You're right! I updated the code even if I'll comment it out since we now use the scheduler plugin 👍